### PR TITLE
OSS Move Update

### PR DESCRIPTION
**Pull request by jeffrey-opdam**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Team-Services-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Updated the file, preparing to move to OSS.
- 
- 

@ALM-Rangers/VSTS-Bot

### CODE REVIEW

File: ThirdPartyNotices.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
-Visual Studio Team Services Bot incorporates components from the projects listed below. Microsoft licenses these components to you under Microsoft�s licensing terms for the Visual Studio Team Services Bot.  The original copyright notices and the licenses under which Microsoft received such components are set forth below for informational purposes. Microsoft reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
+Visual Studio Team Services Bot incorporates components from the projects listed below. ALM | DevOps Ranger Contributors these components to you under Microsoft�s licensing terms for the Visual Studio Team Services Bot.  The original copyright notices and the licenses under which Microsoft received such components are set forth below for informational purposes. Microsoft reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
```
Comment (wpschaub) [inline ThirdPartyNotices.txt]: @jeffrey-opdam It should read "ALM | DevOps Ranger Contributors **license** ..."

Comment (wpschaub) [inline ThirdPartyNotices.txt]: 1. ALM | DevOps Ranger Contributors **license** these components to you under **ALM | DevOps Ranger's** licensing ...
2. The original copyright notices and the licenses under which **ALM | DevOps Rangers** received ...

**Comment by jeffrey-opdam** [comment]
Second try

https://github.com/ALM-Rangers/VSTS-Bot/pull/87

File: ThirdPartyNotices.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
 Do Not Translate or Localize
 
-Visual Studio Team Services Bot incorporates components from the projects listed below. ALM | DevOps Ranger Contributors these components to you under Microsoft�s licensing terms for the Visual Studio Team Services Bot.  The original copyright notices and the licenses under which Microsoft received such components are set forth below for informational purposes. Microsoft reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
+Visual Studio Team Services Bot incorporates components from the projects listed below. ALM | DevOps Ranger Contributors license these components to you under ALM | DevOps Ranger�s licensing terms for the Visual Studio Team Services Bot.  The original copyright notices and the licenses under which ALM | DevOps Rangers received such components are set forth below for informational purposes. ALM | DevOps Rangers reserves all rights not expressly granted herein, whether by implication, estoppel or otherwise.
```
Comment (wpschaub) [inline ThirdPartyNotices.txt]: Sorry, can you change "ALM | DevOps Rangers reserves ..." to "ALM | DevOps Rangers reserve"? Drop s on reserves as Rangers is already a multiple.

**Comment by jeffrey-opdam** [comment]
Updated reserves to reserve

File: ThirdPartyNotices.txt (hunk 2 above)
Comment (jeffrey-opdam) [reply ThirdPartyNotices.txt]: Done

**Pull request merged**